### PR TITLE
instance_norm + configs.html

### PR DESCRIPTION
**Pull request by kimdahyeon977**
헷갈리는 용어들
1. Fast stylization : 빠른 스타일화
2. representation : 표현? /특징?
3. layer 레이어
4. weight decay 가중치 감쇠 
5. learning rate 학습률

### CODE REVIEW

**Comment by minyoungci** [comment]
representation 은 표현 / 표현력 같습니다 ! 
wieght decay는 그대로 써도 되지 않을까요 ?! 
layer는 저도 그대로 레이어라고 표현했습니다 ! 

고생하셨어요 !!

**Comment by russellgeum** [comment]
2. representation은 "표현"

**Comment by movie5** [comment]
감사합니다 :)
